npm/extensions.bzl

```diff
@@ -215,11 +215,6 @@ def _npm_translate_lock_attrs():
     attrs["name"] = attr.string()
     attrs["lifecycle_hooks_exclude"] = attr.string_list(default = [])
     attrs["lifecycle_hooks_no_sandbox"] = attr.bool(default = True)
-
-    # Note, pnpm_version can't be a tuple here, so we can't accept the integrity hash.
-    # This is okay since you can just call the pnpm module extension below first.
-    # TODO(2.0): drop pnpm_version from this module extension
-    attrs["pnpm_version"] = attr.string(default = LATEST_PNPM_VERSION)
```

Comment:
@gregmagolan this is technically a breaking change, however previously it was a noop and didn't do anything. So this might break users builds, but exposes the fact that setting this attribute was a pointless noop leading to confusion.

WDYT?

Reply:
What we typically do in this case is `fail(...)` with a helpful migration message. For users that have it set and think its working this will be better than a bazel failure with "unrecognized parameter"

Reply:
Done, see https://github.com/aspect-build/rules_js/compare/30b5068f59d2895f3c06fc7b94a75efb2fccaf2d..eb9f39f3ce48c2a02b4f8448364734918f7c8938